pkg/util/querylimits/propagation.go

```diff
@@ -20,10 +21,12 @@ const (
 // NOTE: we use custom `model.Duration` instead of standard `time.Duration` because,
 // to support user-friendly duration format (e.g: "1h30m45s") in JSON value.
 type QueryLimits struct {
-	MaxQueryLength          model.Duration `json:"maxQueryLength,omitempty"`
-	MaxQueryLookback        model.Duration `json:"maxQueryLookback,omitempty"`
-	MaxEntriesLimitPerQuery int            `json:"maxEntriesLimitPerQuery,omitempty"`
-	QueryTimeout            model.Duration `json:"queryTimeout,omitempty"`
+	MaxQueryLength          model.Duration   `json:"maxQueryLength,omitempty"`
+	MaxQueryLookback        model.Duration   `json:"maxQueryLookback,omitempty"`
+	MaxEntriesLimitPerQuery int              `json:"maxEntriesLimitPerQuery,omitempty"`
+	QueryTimeout            model.Duration   `json:"queryTimeout,omitempty"`
+	MaxQueryBytesRead       flagext.ByteSize `json:"maxQueryBytesRead,omitempty"`
+	MaxQuerierBytesRead     flagext.ByteSize `json:"maxQuerierBytesRead,omitempty"`
```

Comment:
This should not be exposed.

Reply:
Done as of [db98d7c](https://github.com/grafana/loki/pull/8670/commits/db98d7c7af1d3361d13ec7b08d8458f95f60a1a4)